### PR TITLE
VAULT-18172: Part 2 - Port over the bulk GitHub App installation script to Vault Radar

### PR DESCRIPTION
**Pull request by kathytong-hashicorp**
## Description
- Tested against GitHub Enterprise Server 3.13.3
- Validated bulk installation (with pagination) and bulk uninstallation (with pagination) work
- Validated edge cases when user doesn't have permission to install and when user doesn't have permission to uninstall
- Validated basic login against GitHub Enterprise Server works
- Did not validate 2FA, did not implement SSO
- Did not validate GitHub Cloud

Resolves # [VAULT-18172](https://hashicorp.atlassian.net/browse/VAULT-18172)

## How has this been tested?
- Have independently run the two commands below to completion on a GitHub Enterprise Server instance at version 3.13.3 
```
python3 -u main.py 2>&1 | tee -a bulk-install-app-results.txt
python3 -u main.py --uninstall 2>&1 | tee -a bulk-install-app-results.txt
```

### CODE REVIEW

File: bulk-install-app/debug.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+from config import DEBUG
+
+
+def save_debug_info(folder, name, response):
+    if DEBUG:
+        if not os.path.exists(f"{folder}"):
+            os.makedirs(f"{folder}")
+
+        file = open(f"{folder}/{name}-content.html", "w")
```
Comment (kartheek-hc) [inline bulk-install-app/debug.py:11]: Can we use `with open` so we don't need to worry about closing the file when there is a problem writing content?

**Review by kartheek-hc — APPROVED** [review]

File: bulk-install-app/debug.py (hunk 1 above)
Comment (kathytong-hashicorp) [reply bulk-install-app/debug.py]: Done!

File: bulk-install-app/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Requirements
+- [python >= 3.6](https://www.python.org/downloads/)
+- [python package requests](https://pypi.org/project/requests/)
+- [python package python-dotenv](https://pypi.org/project/python-dotenv/)
+- [python package beautifulsoup4](https://pypi.org/project/beautifulsoup4/)
+## Bulk installing a GitHub Checks App across organizations
+1. Navigate to Vault Radar portal, and import the organizations that need to install the GitHub App 
+2. In `hashicorp-guides/vault-radar-demo/bulk-install-app`, generate an environment file `.env`, with the following variables set:
+   - `USERNAME` - the GitHub user's username
```
[the remark this answers is not in the document]
Comment (kathytong-hashicorp) [reply bulk-install-app/README.md]: Done!

File: bulk-install-app/debug.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+def save_debug_info(folder, name, response):
+    if DEBUG:
+        if not os.path.exists(f"{folder}"):
+            os.makedirs(f"{folder}")
+
+        with open(f"{folder}/{name}-content.html", "w") as file:
+            file.write(response.content.decode("utf-8"))
+            file.close()
```
Comment (kartheek-hc) [inline bulk-install-app/debug.py:13]: I don't think we need to call `file.close()` explicitly

**Pull request merged**